src/appMain/sample_policy_manager.py

```diff
@@ -25,12 +25,17 @@
 
 
 def http_header(data):
+# The Content-Length to be sent in the HTTP Request header should be
+# adjusted for additional escape characters added for newline strings
+# The mobile proxy will remove the escape characters after receiving this request.
+    content_length = len(data) - data.count('\\n')
```

Comment:
pretty sure this should just be done for all escape characters
```suggestion
    content_length = len(data) - data.count('\\')
```